packages/webapp/src/elections/components/ongoing-election-components/request-election-meeting-link-button.tsx

```diff
@@ -100,7 +162,9 @@ export const RequestElectionMeetingLinkButton = ({
         }
     };
 
-    return roundMeetingLink ? (
+    return loadingEncryptedData ? (
+        <>Loading Meeeting Link...</>
```

Comment:
Should we make this a disabled, loading button? Also, one too many "e" chars in "Meeeting". If you prefer, I can tweak this UI too.

Reply:
Yep, I will merge as is and let you to polish the waiting button then! :)